WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -41,18 +42,50 @@ final class DashboardViewController: UIViewController {
         return label
     }()
 
+    /// A stack view to display `storeNameLabel` with additional margins
+    ///
+    private lazy var innerStackView: UIStackView = {
+        let view = UIStackView()
+        view.layoutMargins = UIEdgeInsets(top: 0, left: Constants.leadingMargin, bottom: 0, right: 0)
+        view.isLayoutMarginsRelativeArrangement = true
+        return view
+    }()
+
+    /// A stack view for views displayed between the navigation bar and content (e.g. store name subtitle, top banner)
+    ///
     private lazy var stackView: UIStackView = {
         let view = UIStackView()
         view.translatesAutoresizingMaskIntoConstraints = false
         view.backgroundColor = .listForeground
-        view.layoutMargins = UIEdgeInsets(top: 0, left: navigationController?.navigationBar.directionalLayoutMargins.leading ?? 0, bottom: 0, right: 0)
```

Comment:
This line seems to have caused an issue with the margin when app is launched in landscape mode - removing this seems to have fixed the issue. Bravo 🎉